app/controllers/groups_controller.rb

```diff
@@ -61,6 +61,17 @@ def destroy
     end
   end
 
+  def transfer
+    new_namespace ||= Namespace.find_by(id: params.require(:new_namespace_id))
+    if Groups::TransferService.new(@group, current_user).execute(new_namespace)
+      flash[:success] = t('.success')
+      redirect_to group_path(@group)
+    else
+      @error = @group.errors.messages.values.flatten.first
+      render :edit, status: :unprocessable_entity
```

Comment:
**nitpick (blocking):** Can you update the error to use a turbo stream instead of rendering the full page to show the error  message.  I just fixed this issue on the projects transfer page.  You can see the updated code in https://github.com/phac-nml/irida-next/pull/139

Reply:
I am assuming re-rendering the form is okay, please see 110e9c449274c36172d8ee8f77a4ebb6f0b9feec.